tket/tests/test_ControlDecomp.cpp

```diff
@@ -398,6 +405,26 @@ SCENARIO("Test a CnX is decomposed correctly using the Gray code method") {
       correct_matrix(m_size - 2, m_size - 2) = 0;
       correct_matrix(m_size - 1, m_size - 1) = 0;
       REQUIRE(m.isApprox(correct_matrix, ERR_EPS));
+      switch (n) {
+        case 2: {
+          REQUIRE(circ.count_gates(OpType::CX) <= 6);
```

Comment:
Please add `break` statements at the end of each `case`. (As it happens fall-through means we still pass because they're in order, but is unnecessary!)

Reply:
Ahh, sorry, I'll make the change now!